Bricks/net/http/test.cc

```diff
@@ -422,6 +422,8 @@ TEST(PosixHTTPServerTest, ChunkedLargeBodyManyChunks) {
   t.join();
 }
 
+// This test sometimes fails on Apple on Travis. -- D.K.
+#ifndef CURRENT_APPLE
```

Comment:
Maybe replace the disabled tests with something that will print that they were disabled, to make this obvious and explicit?